src/connections/destinations/catalog/algolia-insights/index.md

```diff
@@ -126,17 +103,4 @@ analytics.track('Order Completed', {
 Track calls will be sent to Algolia as a `track` event, and appear in your Click Analytics, A/B Testing and Personalization dashboard.
```

Comment:
This is a strange line. Can you make heads or tails of it? Insights events are not sent to Algolia "as a track event". They are sent to us as… "events". They have an `eventType` field, but those are one of 'view', 'click' or 'conversion'. I think this needs to be reworded or removed.

Reply:
Yeah this was in the original documentation, I thought it was just me not fully understanding it. I'll just remove this entirely